### PR TITLE
Jw oauth

### PR DESCRIPTION
**Pull request by jaylupus**

### CODE REVIEW

File: server/db/models/user.js [hunk 1]
```diff
@@ -14,18 +14,6 @@ var schema = new mongoose.Schema({
     salt: {
         type: String
     },
-    twitter: {
```
Comment (omcrobbie) [inline server/db/models/user.js:18]: why did you take out this part?

File: server/env/development.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-module.exports = {
```
Comment (omcrobbie) [inline server/env/development.js:1]: is this part of .gitignore now?

Comment (jaylupus) [inline server/env/development.js:1]: yes

File: server/db/models/user.js (hunk 1 above)
Comment (jaylupus) [inline server/db/models/user.js:18]: looked like we weren't using it

**Pull request merged**